### ISSUE TITLE
FEATURE: custom features per language

### ISSUE THREAD

**Issue by rfc2119** [pacollins/hugo-future-imperfect-slim#125]
# Feature Request

Coming from #120 , I wanted to assign a font AND other styling elements  to a language other than English, so that the generated HTML or CSS is different per language

## Solution

I do not know how to approach this, and if the feature is specific to the theme or Hugo in general,  but a simple CSS per language would ideally work

## Alternatives
An example of such styling elements is support for right-to-left languages. The [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction) property for CSS or the [dir](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) global HTML attribute achieves this. What I ended up with is modifiying `static/css/add-on.css` to include:

```css
h1,
h2,
h3,
h4,
h5,
h6, p, a, del, mark, ul, li,  table, tr, td,{
  direction: rtl;
  letter-spacing: unset;
}
```
Obviously, this breaks any English content

**Comment by pacollins** [comment]
There is actually a way to treat any file as a Hugo template. If I can find this, we might be able to do the CSS under that language's i18n.

**Comment by pacollins** [comment]
I'm not sure exactly what you are looking for, but the new PR should be a move in the right direction.

**Comment by rfc2119** [comment]
Thank you for your quick response Patrick!

> There is actually a way to treat any file as a Hugo template. If I can find this, we might be able to do the CSS under that language's i18n.

I am fairly new to Hugo and I am not familiar with [Hugo templates](https://gohugo.io/templates/introduction/). I can't help you there, but I found [this](https://gohugo.io/templates/files/) from a community post on the forums.




> I'm not sure exactly what you are looking for, but the new PR should be a move in the right direction.

Regarding the `letter-spacing` property, some languages does not tolerate spacing between letters, as letters themselves are drawn in "continuous" form, like नेपाल. I added "features" as title of the issue to hint about any other style elements that a language might need

**Comment by pacollins** [comment]
So can you clarify what you are looking for and/or review the PR #126? I am not sure of the specifics *you* are trying to accomplish. What languages. What CSS.

When I further investigated using Hugo Templates for CSS, it seems that you can't use i18n, which makes sense actually. What we can do is load specific CSS files at the end of the styles block based on the language, which is how I set up the PR.

**Comment by rfc2119** [comment]
Hello again, I'm sorry for my late reply.

I have uncommented my manual changes in `add-on.css` and checked out your branch. I think the line
https://github.com/pacollins/hugo-future-imperfect-slim/blob/4d095547178604bfb243aca0a2c4f34c4255b0c9/layouts/partials/head.html#L20
does not evaluated to true, because I see no _rtl.css_ file included (I've copied it to _/static/css_). How do I activate it ?



> I am not sure of the specifics _you_ are trying to accomplish. What languages. What CSS

I have a couple of pages in Arabic for which the aforementioned CSS properties is essential
